I/IOAPI/build_tarballs.jl

```diff
@@ -0,0 +1,134 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "IOAPI"
+version = v"3.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cjcoats/ioapi-3.2.git", "ef5d5f4e112c249b593b19426421f25d79ae094b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add tcsh # build script is in csh
+
+cd $WORKSPACE/srcdir/ioapi-3.2/ioapi/
+install_license LICENSE
```

Comment:
Meh, wrong directory:
```suggestion
install_license ../LICENSE
```